app/src/main/java/com/github/neighbortrader/foodboardapp/requests/GroceryCategoryHandler.java

```diff
@@ -0,0 +1,92 @@
+package com.github.neighbortrader.foodboardapp.requests;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.github.neighbortrader.foodboardapp.clientmodel.Grocery;
+import com.github.neighbortrader.foodboardapp.clientmodel.Offer;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.Getter;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class GroceryCategoryHandler extends AsyncRequest<Grocery> {
+    public static String TAG = GroceryCategoryHandler.class.getSimpleName();
+
+    @Getter
+    private static ArrayList<String> groceriesCategories;
+
+    protected GroceryCategoryHandler(Context context, OnEventListener callback) {
+        super(context, callback);
+    }
+
+    public static GroceryCategoryHandler builder(RequestTyps requestTyps, Context context, OnEventListener callback, Offer... offers) {
```

Comment:
Das gehört eigentlich in eine Factory-Klasse. Allerdings gab es da ja wie gesagt ein paar Probleme. Deswegen ist das erstmal nach hinten verschoben. #15